lib/ruby/signature/scaffold/rbi.rb

```diff
@@ -498,6 +498,10 @@ def const_to_name(node)
               BuiltinNames::Hash.name
             when "T::Range"
               BuiltinNames::Range.name
+            when "T::Enumerator"
+              BuiltinNames::Enumerator.name
+            when "T::Enumerable"
```

Comment:
FYI, this is also missing `T::Set`, though it's not used very much in the stdlib/core sigs. The only example I can find is https://github.com/sorbet/sorbet/blob/2275752e51604acfb79b30a0a96debc996c089d9/rbi/stdlib/set.rbi#L13